### ISSUE TITLE
Fresh airsonic install - media folders not found

### ISSUE THREAD

**Issue by tonyshangrila** [airsonic/airsonic#1070]
## Problem description

Fresh install of airsonic 10.3.1 on Ubuntu 18.04 LTS.  After following standalone installation guide, airsonic cannot see ANY directories/folders on the localhost server.

### Steps to reproduce

1. Install airsonic on Ubuntu
2. log in as admin
3. try to add media folders
4. receive "folder not found" error

## System information

 * **Airsonic version**: 10.3.1
 * **Operating system**: Ubuntu 18.04.2
 * **Java version**: 11.0.3
 * **Proxy server**: Apache 2.4.39
 * **Client**: Chrome 74

## Additional notes

Exact same setup was working fine under previous version, but ended up completely blowing away my db and other settings in an effort to troubleshoot.  Have tried setting media folders to 0777 file permissions with no change, airsonic still can't see folders.  Also tried to run subsonic as a more privileged user with no change.  Assume there must be some other permission issue that I'm not seeing, but currently at a loss.
**MORE INFO, 5/22 -** In order to test whether airsonic could see any part of the file system at all, I tried to use `/` as the media folder.  That one it could see, and scanning found hundreds of thousands of files.

**Comment by jvoisin** [comment]
Have you checked your logs?

**Comment by tonyshangrila** [comment]
I don't see anything unusual in the airsonic.log.  Here's an excerpt from the last time I started the service and tried to add some folders:

> 2019-05-21 22:33:48.988  INFO --- org.airsonic.player.Application          : Starting Application v10.3.1-RELEASE on echobase with PID 2388 (/var/airsonic/airsonic.war started by airsonic in /)
2019-05-21 22:33:48.998  INFO --- org.airsonic.player.Application          : The following profiles are active: legacy
2019-05-21 22:33:52.990  INFO --- o.a.p.service.SettingsService            : Java: 11.0.3, OS: Linux
2019-05-21 22:33:53.426  INFO --- org.airsonic.player.Application          : Detected Tomcat web server
2019-05-21 22:34:02.053  INFO --- l.executor.jvm.JdbcExecutor              : SELECT COUNT(*) FROM DATABASECHANGELOGLOCK
2019-05-21 22:34:02.065  INFO --- l.executor.jvm.JdbcExecutor              : SELECT COUNT(*) FROM DATABASECHANGELOGLOCK
2019-05-21 22:34:02.084  INFO --- l.l.StandardLockService                  : Successfully acquired change log lock
2019-05-21 22:34:04.160  INFO --- l.executor.jvm.JdbcExecutor              : SELECT MD5SUM FROM DATABASECHANGELOG WHERE MD5SUM IS NOT NULL
2019-05-21 22:34:04.167  INFO --- l.executor.jvm.JdbcExecutor              : SELECT COUNT(*) FROM DATABASECHANGELOG
2019-05-21 22:34:04.169  INFO --- l.c.StandardChangeLogHistoryService      : Reading from DATABASECHANGELOG
2019-05-21 22:34:04.170  INFO --- l.executor.jvm.JdbcExecutor              : SELECT * FROM DATABASECHANGELOG ORDER BY DATEEXECUTED ASC, ORDEREXECUTED ASC
2019-05-21 22:34:04.383  INFO --- l.l.StandardLockService                  : Successfully released change log lock
2019-05-21 22:34:04.385  INFO --- l.executor.jvm.JdbcExecutor              : SELECT COUNT(*) FROM DATABASECHANGELOGLOCK
2019-05-21 22:34:04.386  INFO --- l.executor.jvm.JdbcExecutor              : SELECT COUNT(*) FROM DATABASECHANGELOGLOCK
2019-05-21 22:34:04.388  INFO --- l.executor.jvm.JdbcExecutor              : SELECT LOCKED FROM DATABASECHANGELOGLOCK WHERE ID=1
2019-05-21 22:34:04.393  INFO --- l.l.StandardLockService                  : Successfully acquired change log lock
2019-05-21 22:34:04.405  INFO --- l.executor.jvm.JdbcExecutor              : SELECT MD5SUM FROM DATABASECHANGELOG WHERE MD5SUM IS NOT NULL
2019-05-21 22:34:04.408  INFO --- l.executor.jvm.JdbcExecutor              : SELECT COUNT(*) FROM DATABASECHANGELOG
2019-05-21 22:34:04.409  INFO --- l.c.StandardChangeLogHistoryService      : Reading from DATABASECHANGELOG
2019-05-21 22:34:04.410  INFO --- l.executor.jvm.JdbcExecutor              : SELECT * FROM DATABASECHANGELOG ORDER BY DATEEXECUTED ASC, ORDEREXECUTED ASC
2019-05-21 22:34:04.430  INFO --- l.l.StandardLockService                  : Successfully released change log lock
2019-05-21 22:34:04.620  INFO --- o.airsonic.player.dao.PlayerDao          : Deleted 1 player(s) that haven't been used after Fri Mar 22 22:34:04 CDT 2019
2019-05-21 22:34:04.843  INFO --- o.a.p.service.PodcastService             : Automatic Podcast update scheduled to run every 24 hour(s), starting at Tue May 21 22:39:04 CDT 2019
2019-05-21 22:34:04.963  INFO --- o.a.p.s.MediaScannerService              : Automatic media library scanning scheduled to run every 1 day(s), starting at Wed May 22 03:00:00 CDT 2019
2019-05-21 22:34:09.986  INFO --- org.airsonic.player.Application          : Started Application in 23.018 seconds (JVM running for 23.991)
2019-05-21 22:34:29.669  INFO --- o.a.p.f.BootstrapVerificationFilter      : Servlet container: Apache Tomcat/8.5.40
2019-05-21 22:34:43.069  INFO --- o.a.p.service.VersionService             : Resolved local Airsonic version to: 10.3.1-RELEASE
2019-05-21 22:34:43.071  INFO --- o.airsonic.player.dao.PlayerDao          : Created player 1.
2019-05-21 22:34:45.083  WARN --- o.d.impl.DefaultPageNormalizer           : Missing /WEB-INF/web.xml
2019-05-21 22:35:00.434  INFO --- o.a.player.dao.MusicFolderDao            : Created music folder /home/jason/Music
2019-05-21 22:35:00.451  INFO --- o.a.p.s.MediaScannerService              : Automatic media library scanning scheduled to run every 1 day(s), starting at Wed May 22 03:00:00 CDT 2019
2019-05-21 22:37:13.757  INFO --- o.airsonic.player.dao.PlayerDao          : Created player 2.
2019-05-21 22:39:04.842  INFO --- o.a.p.service.PodcastService             : Starting scheduled Podcast refresh.
2019-05-21 22:39:04.844  INFO --- o.a.p.service.PodcastService             : Completed scheduled Podcast refresh.
2019-05-21 22:42:11.331  INFO --- o.a.p.s.MediaScannerService              : Automatic media library scanning scheduled to run every 1 day(s), starting at Wed May 22 03:00:00 CDT 2019
2019-05-21 22:42:23.550  INFO --- o.a.player.dao.MusicFolderDao            : Deleted music folder with ID 3
2019-05-21 22:42:23.552  INFO --- o.a.p.s.MediaScannerService              : Automatic media library scanning scheduled to run every 1 day(s), starting at Wed May 22 03:00:00 CDT 2019
2019-05-21 23:00:46.269  INFO --- o.a.player.dao.MusicFolderDao            : Created music folder /home/jason/Music
2019-05-21 23:00:46.273  INFO --- o.a.p.s.MediaScannerService              : Automatic media library scanning scheduled to run every 1 day(s), starting at Wed May 22 03:00:00 CDT 2019
2019-05-21 23:00:51.448  INFO --- o.a.player.dao.MusicFolderDao            : Deleted music folder with ID 4
2019-05-21 23:00:51.454  INFO --- o.a.p.s.MediaScannerService              : Automatic media library scanning scheduled to run every 1 day(s), starting at Wed May 22 03:00:00 CDT 2019
2019-05-21 23:01:45.202  INFO --- o.a.player.dao.MusicFolderDao            : Created music folder /var/airsonic/music
2019-05-21 23:01:45.205  INFO --- o.a.p.s.MediaScannerService              : Automatic media library scanning scheduled to run every 1 day(s), starting at Wed May 22 03:00:00 CDT 2019
2019-05-21 23:01:51.971  INFO --- o.a.p.s.MediaScannerService              : Automatic media library scanning scheduled to run every 1 day(s), starting at Wed May 22 03:00:00 CDT 2019
2019-05-21 23:01:58.488  INFO --- o.a.p.s.MediaScannerService              : Automatic media library scanning scheduled to run every 1 day(s), starting at Wed May 22 03:00:00 CDT 2019
2019-05-21 23:02:03.501  INFO --- o.a.p.s.MediaScannerService              : Automatic media library scanning scheduled to run every 1 day(s), starting at Wed May 22 03:00:00 CDT 2019
2019-05-21 23:02:07.445  INFO --- o.a.player.dao.MusicFolderDao            : Deleted music folder with ID 5
2019-05-21 23:02:07.448  INFO --- o.a.p.s.MediaScannerService              : Automatic media library scanning scheduled to run every 1 day(s), starting at Wed May 22 03:00:00 CDT 2019
2019-05-22 03:00:00.448  INFO --- o.a.p.s.MediaScannerService              : Starting to scan media library.
2019-05-22 03:00:00.537  INFO --- o.a.p.s.MediaScannerService              : Scanned media library with 0 entries.
2019-05-22 03:00:00.537  INFO --- o.a.p.s.MediaScannerService              : Marking non-present files.
2019-05-22 03:00:00.539  INFO --- o.a.p.s.MediaScannerService              : Marking non-present artists.
2019-05-22 03:00:00.555  INFO --- o.a.p.s.MediaScannerService              : Marking non-present albums.
2019-05-22 03:00:00.574  INFO --- o.a.p.s.MediaScannerService              : Completed media library scan.
2019-05-22 03:00:00.613  INFO --- o.a.p.service.PlaylistService            : Starting playlist import.
2019-05-22 03:00:00.613  INFO --- o.a.p.service.PlaylistService            : Completed playlist import.

Not sure which other logs might be relevant.
